pom.xml

```diff
@@ -48,6 +48,12 @@
       <artifactId>ssh-cli-auth</artifactId>
       <version>1.4</version>
     </dependency>
+    <dependency>
+      <groupId>org.bouncycastle</groupId>
+      <artifactId>bcpkix-jdk15on</artifactId>
+      <version>1.58</version>
+      <optional>true</optional>
```

Comment:
We need to build against BouncyCastle, but we do not want to bundle it. We use `SecurityUtils#isBouncyCastleRegistered` to decide at runtime if we should actually load the BouncyCastle-related classes.

Reply:
scope = provided ?